### PR TITLE
Completed array-reduce

### PR DESCRIPTION
**Pull request by K-Bowler**
image

### CODE REVIEW

File: array-reduce/main.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+function accountBalance(total) {
+  return account.reduce((x, y) => {
+    if (x.type) {
+      total = x.amount + y.amount;
+    } else if (y.type === 'withdrawal') {
+      total = total - y.amount;
+    } else if (y.type === 'deposit') {
+      total = total + y.amount;
+    }
```
Comment (scbowler) [inline array-reduce/main.js]: You don't need the first `if` statement here, just need to check for `withdrawal` or `deposit`:
```JAVASCRIPT
if (y.type === 'deposit') {
  total = total + y.amount;
} else {
  total = total - y.amount;
}
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just one update needed. Let me know if you have any questions.

File: array-reduce/main.js (hunk 1 above)
Comment (K-Bowler) [reply array-reduce/main.js]: image

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**